src/indentPattern.ts

```diff
@@ -0,0 +1,18 @@
+export function decreaseIndentPattern() {
+  return /((^\s*end\b\s*)\b(if|while|for|match|try|\w+)$|(^(.*\*\/)?\s*\}.*)$)/;
+}
+export function increaseIndentPattern() {
+  // substituted the following "old if syntax" regex from smarter's repo
```

Comment:
No need for the comment, it's something related to this PR, but might be confusing in the future, especially someone might not know who is `smarter`